api/server/sliceModel.go

```diff
@@ -2,6 +2,7 @@ package server
 
 import (
 	"fmt"
+	"log"
```

Comment:
consider using the golog package. Then you can use golog.Error, golog.Info etc. respcts the LOG_LEVEL env parameter